gulpfile.babel.js

```diff
@@ -154,10 +160,52 @@ function litmus() {
   var cdnURL = CONFIG.aws.url;
 
   return gulp.src('dist/**/*.html')
-    .pipe($.replace(/=('|")(\/?assets\/img)/g, "=$1"+ cdnURL +"/"+ CONFIG.common.subject))
+    .pipe($.replace(/=('|")(\/?images)/g, "=$1"+ cdnURL +"/"+ CONFIG.common.subject))
     .pipe($.litmus(CONFIG.litmus));
 }
 
+// Copy and compress into Zip
+function zip() {
+  var dist = 'dist';
+  var ext = '.html';
+
+  function getHtmlFiles(dir) {
+    return fs.readdirSync(dir)
+      .filter(function(file) {
+        var fileExt = path.join(dir, file);
+        var isHtml = path.extname(fileExt) == ext;
+        return fs.statSync(fileExt).isFile() && isHtml;
+      });
+  }
+
+  var htmlFiles = getHtmlFiles(dist);
+
+  var moveTasks = htmlFiles.map(function(file){
+    var sourcePath = path.join(dist, file);
+    var fileName = path.basename(sourcePath, ext);
+
+    var moveHTML = gulp.src(sourcePath)
+      .pipe($.rename(function (path) {
+        path.dirname = fileName;
+        return path;
+      }));
+
+    var moveImages = gulp.src(sourcePath)
+      .pipe($.htmlSrc({ selector: 'img'}))
```

Comment:
I'd love this to work. In my workflow I'm seeing gulp crash at this point. 

![npm run zip](https://www.evernote.com/shard/s495/sh/b43c003b-7431-47e4-8208-2748648d6fef/605187c659819eab/res/6fbd6d6c-afbf-4d45-ae47-b047ae030684/skitch.png)

Two console logs here to see where the crash occurs. See the first one succeed in the output above.

![gulpfile](https://www.evernote.com/shard/s495/sh/a825360f-e902-4d1e-8079-f5a15b86eb09/0914c8cce737c6ea/res/10a71336-93ad-4c57-bd0d-7e4b8cb22ca3/skitch.png)

Never reach the second log, so the crash occurs in the `gulp.src` or `$.htmlSrc` functions?

![error output](https://www.evernote.com/shard/s495/sh/3562e160-1b29-45b9-8999-aded067c0eb2/ed83cb66ba681f2e/res/d51c5362-e009-4165-82a9-d093d40e5e8b/skitch.png)

Not sure where to file an issue for this. This commit seemed like the most logical place.